### DISCUSSION TITLE
Magic carpet ride badge for full year of consecutive commits

### DISCUSSION THREAD

**Discussion by pshemekpi** [community/community#30298] [Other Feature Feedback, Questions, & Ideas]
Hi,
I'm not sure it this is the right place for it, but I thought about the idea of new badge for full year of consecutive commits:

**Magic carpet ride**

Reasoning:
Full year of commits is quite an achievement and kind of looks like a knitted carpet with different hues of threads on people profile (apart also from looking like bathroom tiles). 
Also the song is cool.
If there is another place that this proposal suits better I'd be glad if you let me know :)
Or maybe I've missed something and there is a proposal like that?

Thanks!

**Comment by 0dminnimda** [comment]
People can use some special tools to mangle with their previous commits to achieve this, and it's not so difficult to do
I would prefer not to encourage this behavior by adding such an achievement
